hmda/src/main/scala/hmda/model/institution/Institution.scala

```diff
@@ -0,0 +1,37 @@
+package hmda.model.institution
+
+object Institution {
+  def empty: Institution = Institution(
+    2018,
+    None,
+    None,
+    None,
+    None,
+    None,
+    None,
+    None,
+    Respondent.empty,
+    Parent.empty,
+    None,
+    None,
+    TopHolder.empty,
+    false
+  )
+}
+
+case class Institution(
+    activityYear: Int,
+    LEI: Option[String],
+    agency: Option[Agency],
+    institutionType: Option[InstitutionType],
+    institutionId_2017: Option[String],
+    taxId: Option[String],
+    rssd: Option[String],
+    emailDomain: Option[String],
```

Comment:
Are we allowing only one email domain per institution?

Reply:
Actually, I don't know. 
@keileyo Should we allow several email domains per institution?

Reply:
yes, we should allow multiple domains per institution @keileyo